<!doctype html><html lang=en dir=ltr class=scroll-smooth data-default-appearance=light data-auto-appearance=true><head><meta charset=utf-8><meta http-equiv=content-language content="en"><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=X-UA-Compatible content="ie=edge"><title>DPLM-2: A Multimodal Diffusion Protein Language Model &#183; AI Paper Reviews by AI</title>
<meta name=title content="DPLM-2: A Multimodal Diffusion Protein Language Model &#183; AI Paper Reviews by AI"><meta name=description content="DPLM-2: a new multimodal model revolutionizes protein design by simultaneously generating protein sequences and 3D structures, surpassing existing methods in accuracy and diversity."><meta name=keywords content="ğŸ”– 24-10-17,ğŸ¤— 24-10-21,"><link rel=canonical href=https://deep-diver.github.io/ai-paper-reviewer/paper-reviews/2410.13782/><link type=text/css rel=stylesheet href=/ai-paper-reviewer/css/main.bundle.min.67eb0befb1fb57e6528684f3a2ff6e0606a82100504705b66488d54a9f3fda9c2605c9bf3b70163208427288637a7b4d1a168384c1115e417458eae6b44b329c.css integrity="sha512-Z+sL77H7V+ZShoTzov9uBgaoIQBQRwW2ZIjVSp8/2pwmBcm/O3AWMghCcohjentNGhaDhMERXkF0WOrmtEsynA=="><script type=text/javascript src=/ai-paper-reviewer/js/appearance.min.516a16745bea5a9bd011138d254cc0fd3973cd55ce6e15f3dec763e7c7c2c7448f8fe7b54cca811cb821b0c7e12cd161caace1dd794ac3d34d40937cbcc9ee12.js integrity="sha512-UWoWdFvqWpvQERONJUzA/TlzzVXObhXz3sdj58fCx0SPj+e1TMqBHLghsMfhLNFhyqzh3XlKw9NNQJN8vMnuEg=="></script><script defer type=text/javascript id=script-bundle src=/ai-paper-reviewer/js/main.bundle.min.efbf3b6b987689fffaf2d7b73173d2690c0279a04d444b0537a77d7f4ff6e6d493445400cb0cf56bc0f0f123e19f15394e63cae34e67f069bd013dd5c73df56e.js integrity="sha512-7787a5h2if/68te3MXPSaQwCeaBNREsFN6d9f0/25tSTRFQAywz1a8Dw8SPhnxU5TmPK405n8Gm9AT3Vxz31bg==" data-copy data-copied></script><script src=/ai-paper-reviewer/lib/zoom/zoom.min.37d2094687372da3f7343a221a470f6b8806f7891aa46a5a03966af7f0ebd38b9fe536cb154e6ad28f006d184b294525a7c4054b6bbb4be62d8b453b42db99bd.js integrity="sha512-N9IJRoc3LaP3NDoiGkcPa4gG94kapGpaA5Zq9/Dr04uf5TbLFU5q0o8AbRhLKUUlp8QFS2u7S+Yti0U7QtuZvQ=="></script><link rel=apple-touch-icon sizes=180x180 href=/ai-paper-reviewer/apple-touch-icon.png><link rel=icon type=image/png sizes=32x32 href=/ai-paper-reviewer/favicon-32x32.png><link rel=icon type=image/png sizes=16x16 href=/ai-paper-reviewer/favicon-16x16.png><link rel=manifest href=/ai-paper-reviewer/site.webmanifest><meta property="og:url" content="https://deep-diver.github.io/ai-paper-reviewer/paper-reviews/2410.13782/"><meta property="og:site_name" content="AI Paper Reviews by AI"><meta property="og:title" content="DPLM-2: A Multimodal Diffusion Protein Language Model"><meta property="og:description" content="DPLM-2: a new multimodal model revolutionizes protein design by simultaneously generating protein sequences and 3D structures, surpassing existing methods in accuracy and diversity."><meta property="og:locale" content="en"><meta property="og:type" content="article"><meta property="article:section" content="paper-reviews"><meta property="article:published_time" content="2024-10-17T00:00:00+00:00"><meta property="article:modified_time" content="2024-10-17T00:00:00+00:00"><meta property="article:tag" content="ğŸ”– 24-10-17"><meta property="article:tag" content="ğŸ¤— 24-10-21"><meta property="og:image" content="https://deep-diver.github.io/ai-paper-reviewer/paper-reviews/2410.13782/cover.png"><meta name=twitter:card content="summary_large_image"><meta name=twitter:image content="https://deep-diver.github.io/ai-paper-reviewer/paper-reviews/2410.13782/cover.png"><meta name=twitter:title content="DPLM-2: A Multimodal Diffusion Protein Language Model"><meta name=twitter:description content="DPLM-2: a new multimodal model revolutionizes protein design by simultaneously generating protein sequences and 3D structures, surpassing existing methods in accuracy and diversity."><script type=application/ld+json>[{"@context":"https://schema.org","@type":"Article","articleSection":"Posts","name":"DPLM-2: A Multimodal Diffusion Protein Language Model","headline":"DPLM-2: A Multimodal Diffusion Protein Language Model","abstract":"DPLM-2: a new multimodal model revolutionizes protein design by simultaneously generating protein sequences and 3D structures, surpassing existing methods in accuracy and diversity.","inLanguage":"en","url":"https:\/\/deep-diver.github.io\/ai-paper-reviewer\/paper-reviews\/2410.13782\/","author":{"@type":"Person","name":"AI Paper Reviews by AI"},"copyrightYear":"2024","dateCreated":"2024-10-17T00:00:00\u002b00:00","datePublished":"2024-10-17T00:00:00\u002b00:00","dateModified":"2024-10-17T00:00:00\u002b00:00","keywords":["ğŸ”– 24-10-17","ğŸ¤— 24-10-21"],"mainEntityOfPage":"true","wordCount":"2200"}]</script><meta name=author content="AI Paper Reviews by AI"><link href=https://github.com/deep-diver/paper-reviewer/ rel=me><link href=https://twitter.com/algo_diver/ rel=me><script src=/ai-paper-reviewer/lib/jquery/jquery.slim.min.b0dca576e87d7eaa5850ae4e61759c065786cdb6489d68fcc82240539eebd5da522bdb4fda085ffd245808c8fe2acb2516408eb774ef26b5f6015fc6737c0ea8.js integrity="sha512-sNylduh9fqpYUK5OYXWcBleGzbZInWj8yCJAU57r1dpSK9tP2ghf/SRYCMj+KsslFkCOt3TvJrX2AV/Gc3wOqA=="></script><script defer src=/ai-paper-reviewer/lib/typeit/typeit.umd.1b3200cb448f5cd1f548f2781452643d3511a43584b377b82c03a58055da4fdb7bc8f6c6c2ce846480c7677ff25bfd0d75f15823c09443ab18e0fd2cad792587.js integrity="sha512-GzIAy0SPXNH1SPJ4FFJkPTURpDWEs3e4LAOlgFXaT9t7yPbGws6EZIDHZ3/yW/0NdfFYI8CUQ6sY4P0srXklhw=="></script><script defer src=/ai-paper-reviewer/lib/packery/packery.pkgd.min.js integrity></script><script type=text/javascript src=/ai-paper-reviewer/js/shortcodes/gallery.min.9b4cb28f931ed922c26fb9b2510c2debb370f6a63305050c2af81740b2919883715e24efbbdf3a081496718ec751df3a72729d4d0bc71d6071297563a97ce1ee.js integrity="sha512-m0yyj5Me2SLCb7myUQwt67Nw9qYzBQUMKvgXQLKRmINxXiTvu986CBSWcY7HUd86cnKdTQvHHWBxKXVjqXzh7g=="></script><meta name=theme-color></head><body class="flex flex-col h-screen px-6 m-auto text-lg leading-7 max-w-7xl bg-neutral text-neutral-900 dark:bg-neutral-800 dark:text-neutral sm:px-14 md:px-24 lg:px-32 scrollbar-thin scrollbar-track-neutral-200 scrollbar-thumb-neutral-400 dark:scrollbar-track-neutral-800 dark:scrollbar-thumb-neutral-600"><div id=the-top class="absolute flex self-center"><a class="px-3 py-1 text-sm -translate-y-8 rounded-b-lg bg-primary-200 focus:translate-y-0 dark:bg-neutral-600" href=#main-content><span class="font-bold text-primary-600 ltr:pr-2 rtl:pl-2 dark:text-primary-400">&darr;</span>Skip to main content</a></div><div class=min-h-[148px]></div><div class="fixed inset-x-0 pl-[24px] pr-[24px]" style=z-index:100><div id=menu-blur class="absolute opacity-0 inset-x-0 top-0 h-full single_hero_background nozoom backdrop-blur-2xl shadow-2xl"></div><div class="relative max-w-[64rem] ml-auto mr-auto"><div style=padding-left:0;padding-right:0;padding-top:2px;padding-bottom:3px class="main-menu flex items-center justify-between px-4 py-6 sm:px-6 md:justify-start space-x-3"><div class="flex flex-1 items-center justify-between"><nav class="flex space-x-3"><a href=/ai-paper-reviewer/ class="text-base font-medium text-gray-500 hover:text-gray-900">AI Paper Reviews by AI</a></nav><nav class="hidden md:flex items-center space-x-5 md:ml-12 h-12"><a href=/ai-paper-reviewer/paper-reviews/ class="flex items-center text-gray-500 hover:text-primary-600 dark:hover:text-primary-400"><p class="text-base font-medium" title>Paper Reviews</p></a><a href=/ai-paper-reviewer/tags/ class="flex items-center text-gray-500 hover:text-primary-600 dark:hover:text-primary-400"><p class="text-base font-medium" title>Tags</p></a><a href=https://github.com/deep-diver/paper-reviewer/ target=_blank class="flex items-center text-gray-500 hover:text-primary-600 dark:hover:text-primary-400"><span><span class="relative block icon"><svg viewBox="0 0 496 512"><path fill="currentcolor" d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6.0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6.0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3.0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1.0-6.2-.3-40.4-.3-61.4.0.0-70 15-84.7-29.8.0.0-11.4-29.1-27.8-36.6.0.0-22.9-15.7 1.6-15.4.0.0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5.0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9.0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4.0 33.7-.3 75.4-.3 83.6.0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6.0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9.0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg></span></span><p class="text-base font-medium" title></p></a><a href=https://twitter.com/algo_diver/ target=_blank class="flex items-center text-gray-500 hover:text-primary-600 dark:hover:text-primary-400"><span><span class="relative block icon"><svg viewBox="0 0 512 512"><path fill="currentcolor" d="M459.37 151.716c.325 4.548.325 9.097.325 13.645.0 138.72-105.583 298.558-298.558 298.558-59.452.0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055.0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421.0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391.0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04.0-57.828 46.782-104.934 104.934-104.934 30.213.0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"/></svg></span></span><p class="text-base font-medium" title></p></a><button id=search-button aria-label=Search class="text-base hover:text-primary-600 dark:hover:text-primary-400" title>
<span class="relative block icon"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="search" class="svg-inline--fa fa-search fa-w-16" role="img" viewBox="0 0 512 512"><path fill="currentcolor" d="M505 442.7 405.3 343c-4.5-4.5-10.6-7-17-7H372c27.6-35.3 44-79.7 44-128C416 93.1 322.9.0 208 0S0 93.1.0 208s93.1 208 208 208c48.3.0 92.7-16.4 128-44v16.3c0 6.4 2.5 12.5 7 17l99.7 99.7c9.4 9.4 24.6 9.4 33.9.0l28.3-28.3c9.4-9.4 9.4-24.6.1-34zM208 336c-70.7.0-128-57.2-128-128 0-70.7 57.2-128 128-128 70.7.0 128 57.2 128 128 0 70.7-57.2 128-128 128z"/></svg></span></button><div class="ltr:mr-14 rtl:ml-14 flex items-center"><button id=appearance-switcher aria-label="Dark mode switcher" type=button class="text-base hover:text-primary-600 dark:hover:text-primary-400"><div class="flex items-center justify-center dark:hidden"><span class="relative block icon"><svg viewBox="0 0 512 512"><path fill="currentcolor" d="M32 256C32 132.2 132.3 32 255.8 32c11.36.0 29.7 1.668 40.9 3.746 9.616 1.777 11.75 14.63 3.279 19.44C245 86.5 211.2 144.6 211.2 207.8c0 109.7 99.71 193 208.3 172.3 9.561-1.805 16.28 9.324 10.11 16.95C387.9 448.6 324.8 480 255.8 480 132.1 480 32 379.6 32 256z"/></svg></span></div><div class="items-center justify-center hidden dark:flex"><span class="relative block icon"><svg viewBox="0 0 512 512"><path fill="currentcolor" d="M256 159.1c-53.02.0-95.1 42.98-95.1 95.1s41.2 96.9 95.1 96.9 95.1-42.98 95.1-95.1S309 159.1 256 159.1zM509.3 347l-63.2-91.9 63.15-91.01c6.332-9.125 1.104-21.74-9.826-23.72l-109-19.7-19.7-109c-1.975-10.93-14.59-16.16-23.72-9.824L256 65.89 164.1 2.736c-9.125-6.332-21.74-1.107-23.72 9.824L121.6 121.6 12.56 141.3C1.633 143.2-3.596 155.9 2.736 164.1L65.89 256 2.74 347.01c-6.332 9.125-1.105 21.74 9.824 23.72l109 19.7 19.7 109c1.975 10.93 14.59 16.16 23.72 9.824L256 446.1l91.01 63.15c9.127 6.334 21.75 1.107 23.72-9.822l19.7-109 109-19.7C510.4 368.8 515.6 356.1 509.3 347zM256 383.1c-70.69.0-127.1-57.31-127.1-127.1.0-70.69 57.31-127.1 127.1-127.1S383.1 186.2 383.1 256c0 70.7-56.4 127.1-127.1 127.1z"/></svg></span></div></button></div></nav><div class="flex md:hidden items-center space-x-5 md:ml-12 h-12"><span></span>
<button id=search-button-mobile aria-label=Search class="text-base hover:text-primary-600 dark:hover:text-primary-400" title>
<span class="relative block icon"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="search" class="svg-inline--fa fa-search fa-w-16" role="img" viewBox="0 0 512 512"><path fill="currentcolor" d="M505 442.7 405.3 343c-4.5-4.5-10.6-7-17-7H372c27.6-35.3 44-79.7 44-128C416 93.1 322.9.0 208 0S0 93.1.0 208s93.1 208 208 208c48.3.0 92.7-16.4 128-44v16.3c0 6.4 2.5 12.5 7 17l99.7 99.7c9.4 9.4 24.6 9.4 33.9.0l28.3-28.3c9.4-9.4 9.4-24.6.1-34zM208 336c-70.7.0-128-57.2-128-128 0-70.7 57.2-128 128-128 70.7.0 128 57.2 128 128 0 70.7-57.2 128-128 128z"/></svg>
</span></button>
<button id=appearance-switcher-mobile aria-label="Dark mode switcher" type=button class="text-base hover:text-primary-600 dark:hover:text-primary-400" style=margin-right:5px><div class="flex items-center justify-center dark:hidden"><span class="relative block icon"><svg viewBox="0 0 512 512"><path fill="currentcolor" d="M32 256C32 132.2 132.3 32 255.8 32c11.36.0 29.7 1.668 40.9 3.746 9.616 1.777 11.75 14.63 3.279 19.44C245 86.5 211.2 144.6 211.2 207.8c0 109.7 99.71 193 208.3 172.3 9.561-1.805 16.28 9.324 10.11 16.95C387.9 448.6 324.8 480 255.8 480 132.1 480 32 379.6 32 256z"/></svg></span></div><div class="items-center justify-center hidden dark:flex"><span class="relative block icon"><svg viewBox="0 0 512 512"><path fill="currentcolor" d="M256 159.1c-53.02.0-95.1 42.98-95.1 95.1s41.2 96.9 95.1 96.9 95.1-42.98 95.1-95.1S309 159.1 256 159.1zM509.3 347l-63.2-91.9 63.15-91.01c6.332-9.125 1.104-21.74-9.826-23.72l-109-19.7-19.7-109c-1.975-10.93-14.59-16.16-23.72-9.824L256 65.89 164.1 2.736c-9.125-6.332-21.74-1.107-23.72 9.824L121.6 121.6 12.56 141.3C1.633 143.2-3.596 155.9 2.736 164.1L65.89 256 2.74 347.01c-6.332 9.125-1.105 21.74 9.824 23.72l109 19.7 19.7 109c1.975 10.93 14.59 16.16 23.72 9.824L256 446.1l91.01 63.15c9.127 6.334 21.75 1.107 23.72-9.822l19.7-109 109-19.7C510.4 368.8 515.6 356.1 509.3 347zM256 383.1c-70.69.0-127.1-57.31-127.1-127.1.0-70.69 57.31-127.1 127.1-127.1S383.1 186.2 383.1 256c0 70.7-56.4 127.1-127.1 127.1z"/></svg></span></div></button></div></div><div class="-my-2 -mr-2 md:hidden"><label id=menu-button class=block><div class="cursor-pointer hover:text-primary-600 dark:hover:text-primary-400"><span class="relative block icon"><svg viewBox="0 0 448 512"><path fill="currentcolor" d="M0 96C0 78.33 14.33 64 32 64H416c17.7.0 32 14.33 32 32 0 17.7-14.3 32-32 32H32C14.33 128 0 113.7.0 96zM0 256c0-17.7 14.33-32 32-32H416c17.7.0 32 14.3 32 32s-14.3 32-32 32H32c-17.67.0-32-14.3-32-32zM416 448H32c-17.67.0-32-14.3-32-32s14.33-32 32-32H416c17.7.0 32 14.3 32 32s-14.3 32-32 32z"/></svg></span></div><div id=menu-wrapper style=padding-top:5px class="fixed inset-0 z-30 invisible w-screen h-screen m-0 overflow-auto transition-opacity opacity-0 cursor-default bg-neutral-100/50 backdrop-blur-sm dark:bg-neutral-900/50"><ul class="flex space-y-2 mt-3 flex-col items-end w-full px-6 py-6 mx-auto overflow-visible list-none ltr:text-right rtl:text-left max-w-7xl"><li id=menu-close-button><span class="cursor-pointer inline-block align-text-bottom hover:text-primary-600 dark:hover:text-primary-400"><span class="relative block icon"><svg viewBox="0 0 320 512"><path fill="currentcolor" d="M310.6 361.4c12.5 12.5 12.5 32.75.0 45.25C304.4 412.9 296.2 416 288 416s-16.38-3.125-22.62-9.375L160 301.3 54.63 406.6C48.38 412.9 40.19 416 32 416S15.63 412.9 9.375 406.6c-12.5-12.5-12.5-32.75.0-45.25l105.4-105.4L9.375 150.6c-12.5-12.5-12.5-32.75.0-45.25s32.75-12.5 45.25.0L160 210.8l105.4-105.4c12.5-12.5 32.75-12.5 45.25.0s12.5 32.75.0 45.25l-105.4 105.4L310.6 361.4z"/></svg></span></span></li><li class=mt-1><a href=/ai-paper-reviewer/paper-reviews/ class="flex items-center text-gray-500 hover:text-primary-600 dark:hover:text-primary-400"><p class="text-bg font-bg" title>Paper Reviews</p></a></li><li class=mt-1><a href=/ai-paper-reviewer/tags/ class="flex items-center text-gray-500 hover:text-primary-600 dark:hover:text-primary-400"><p class="text-bg font-bg" title>Tags</p></a></li><li class=mt-1><a href=https://github.com/deep-diver/paper-reviewer/ target=_blank class="flex items-center text-gray-500 hover:text-primary-600 dark:hover:text-primary-400"><div><span class="relative block icon"><svg viewBox="0 0 496 512"><path fill="currentcolor" d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6.0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6.0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3.0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1.0-6.2-.3-40.4-.3-61.4.0.0-70 15-84.7-29.8.0.0-11.4-29.1-27.8-36.6.0.0-22.9-15.7 1.6-15.4.0.0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5.0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9.0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4.0 33.7-.3 75.4-.3 83.6.0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6.0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9.0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg></span></div><p class="text-bg font-bg" title></p></a></li><li class=mt-1><a href=https://twitter.com/algo_diver/ target=_blank class="flex items-center text-gray-500 hover:text-primary-600 dark:hover:text-primary-400"><div><span class="relative block icon"><svg viewBox="0 0 512 512"><path fill="currentcolor" d="M459.37 151.716c.325 4.548.325 9.097.325 13.645.0 138.72-105.583 298.558-298.558 298.558-59.452.0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055.0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421.0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391.0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04.0-57.828 46.782-104.934 104.934-104.934 30.213.0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"/></svg></span></div><p class="text-bg font-bg" title></p></a></li></ul></div></label></div></div></div></div><script>window.addEventListener("scroll",function(){var t=window.pageYOffset||document.documentElement.scrollTop||document.body.scrollTop||0,n=document.getElementById("menu-blur");n.style.opacity=t/300})</script><div class="relative flex flex-col grow"><main id=main-content class=grow><article><div id=hero class="h-[150px] md:h-[200px]"></div><div class="fixed inset-x-0 top-0 h-[800px] single_hero_background nozoom" style=background-image:url(/ai-paper-reviewer/paper-reviews/2410.13782/cover_hu5716466103122325410.png)><div class="absolute inset-0 bg-gradient-to-t from-neutral dark:from-neutral-800 to-transparent mix-blend-normal"></div><div class="absolute inset-0 opacity-60 bg-gradient-to-t from-neutral dark:from-neutral-800 to-neutral-100 dark:to-neutral-800 mix-blend-normal"></div></div><div id=background-blur class="fixed opacity-0 inset-x-0 top-0 h-full single_hero_background nozoom backdrop-blur-2xl"></div><script>window.addEventListener("scroll",function(){var t=window.pageYOffset||document.documentElement.scrollTop||document.body.scrollTop||0,n=document.getElementById("background-blur");n.style.opacity=t/300})</script><header id=single_header class="mt-5 max-w-prose"><ol class="text-sm text-neutral-500 dark:text-neutral-400 print:hidden"><li class="inline hidden"><a class="hover:underline decoration-neutral-300 dark:underline-neutral-600" href=/ai-paper-reviewer/>AI Paper Reviews by AI</a><span class="px-1 text-primary-500">/</span></li><li class=inline><a class="hover:underline decoration-neutral-300 dark:underline-neutral-600" href=/ai-paper-reviewer/paper-reviews/>Posts</a><span class="px-1 text-primary-500">/</span></li><li class="inline hidden"><a class="hover:underline decoration-neutral-300 dark:underline-neutral-600" href=/ai-paper-reviewer/paper-reviews/2410.13782/>DPLM-2: A Multimodal Diffusion Protein Language Model</a><span class="px-1 text-primary-500">/</span></li></ol><h1 class="mt-0 text-4xl font-extrabold text-neutral-900 dark:text-neutral">DPLM-2: A Multimodal Diffusion Protein Language Model</h1><div class="mt-1 mb-6 text-base text-neutral-500 dark:text-neutral-400 print:hidden"><div class="flex flex-row flex-wrap items-center"><time datetime=2024-10-17T00:00:00+00:00>17 October 2024</time><span class="px-2 text-primary-500">&#183;</span><span>2200 words</span><span class="px-2 text-primary-500">&#183;</span><span title="Reading time">11 mins</span></div><div class="flex flex-row flex-wrap items-center"><span style=margin-top:.5rem class=mr-2 onclick='window.open("/ai-paper-reviewer/categories/ai-generated/","_self")'><span class=flex style=cursor:pointer><span class="rounded-md border border-primary-400 px-1 py-[1px] text-xs font-normal text-primary-700 dark:border-primary-600 dark:text-primary-400">AI Generated
</span></span></span><span style=margin-top:.5rem class=mr-2 onclick='window.open("/ai-paper-reviewer/tags/-24-10-17/","_self")'><span class=flex style=cursor:pointer><span class="rounded-md border border-primary-400 px-1 py-[1px] text-xs font-normal text-primary-700 dark:border-primary-600 dark:text-primary-400">ğŸ”– 24-10-17
</span></span></span><span style=margin-top:.5rem class=mr-2 onclick='window.open("/ai-paper-reviewer/tags/-24-10-21/","_self")'><span class=flex style=cursor:pointer><span class="rounded-md border border-primary-400 px-1 py-[1px] text-xs font-normal text-primary-700 dark:border-primary-600 dark:text-primary-400">ğŸ¤— 24-10-21</span></span></span></div></div><div class="flex author"><img class="!mt-0 !mb-0 h-24 w-24 rounded-full ltr:mr-4 rtl:ml-4" width=96 height=96 alt="AI Paper Reviews by AI" src=/ai-paper-reviewer/img/avatar_hu14127527184135390686.png><div class=place-self-center><div class="text-[0.6rem] uppercase leading-3 text-neutral-500 dark:text-neutral-400">Author</div><div class="font-semibold leading-6 text-neutral-800 dark:text-neutral-300">AI Paper Reviews by AI</div><div class="text-sm text-neutral-700 dark:text-neutral-400">I am AI, and I review papers in the field of AI</div><div class="text-2xl sm:text-lg"><div class="flex flex-wrap text-neutral-400 dark:text-neutral-500"><a class="px-1 hover:text-primary-700 dark:hover:text-primary-400" href=https://github.com/deep-diver/paper-reviewer/ target=_blank aria-label=Github rel="me noopener noreferrer"><span class="inline-block align-text-bottom"><span class="relative block icon"><svg viewBox="0 0 496 512"><path fill="currentcolor" d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6.0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6.0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3.0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1.0-6.2-.3-40.4-.3-61.4.0.0-70 15-84.7-29.8.0.0-11.4-29.1-27.8-36.6.0.0-22.9-15.7 1.6-15.4.0.0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5.0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9.0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4.0 33.7-.3 75.4-.3 83.6.0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6.0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9.0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
</span></span></a><a class="px-1 hover:text-primary-700 dark:hover:text-primary-400" href=https://twitter.com/algo_diver/ target=_blank aria-label=Twitter rel="me noopener noreferrer"><span class="inline-block align-text-bottom"><span class="relative block icon"><svg viewBox="0 0 512 512"><path fill="currentcolor" d="M459.37 151.716c.325 4.548.325 9.097.325 13.645.0 138.72-105.583 298.558-298.558 298.558-59.452.0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055.0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421.0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391.0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04.0-57.828 46.782-104.934 104.934-104.934 30.213.0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"/></svg></span></span></a></div></div></div></div><div class=mb-5></div></header><section class="flex flex-col max-w-full mt-0 prose dark:prose-invert lg:flex-row"><div class="order-first lg:ml-auto px-0 lg:order-last ltr:lg:pl-8 rtl:lg:pr-8"><div class="toc ltr:pl-5 rtl:pr-5 print:hidden lg:sticky lg:top-[140px]"><details open id=TOCView class="toc-right mt-0 overflow-y-scroll overscroll-contain scrollbar-thin scrollbar-track-neutral-200 scrollbar-thumb-neutral-400 dark:scrollbar-track-neutral-800 dark:scrollbar-thumb-neutral-600 rounded-lg ltr:-ml-5 ltr:pl-5 rtl:-mr-5 rtl:pr-5 hidden lg:block"><summary class="block py-1 text-lg font-semibold cursor-pointer bg-neutral-100 text-neutral-800 ltr:-ml-5 ltr:pl-5 rtl:-mr-5 rtl:pr-5 dark:bg-neutral-700 dark:text-neutral-100 lg:hidden">Table of Contents</summary><div class="min-w-[220px] py-2 border-dotted ltr:-ml-5 ltr:border-l ltr:pl-5 rtl:-mr-5 rtl:border-r rtl:pr-5 dark:border-neutral-600"><nav id=TableOfContents><ul><li><ul><li><a href=#tldr>TL;DR</a><ul><li><a href=#why-does-it-matter>Why does it matter?</a></li><li><a href=#key-takeaways>Key Takeaways</a></li><li><a href=#visual-insights>Visual Insights</a></li></ul></li><li><a href=#more-visual-insights>More visual insights</a></li><li><a href=#full-paper>Full paper</a></li></ul></li></ul></nav></div></details><details class="toc-inside mt-0 overflow-hidden rounded-lg ltr:-ml-5 ltr:pl-5 rtl:-mr-5 rtl:pr-5 lg:hidden"><summary class="py-1 text-lg font-semibold cursor-pointer bg-neutral-100 text-neutral-800 ltr:-ml-5 ltr:pl-5 rtl:-mr-5 rtl:pr-5 dark:bg-neutral-700 dark:text-neutral-100 lg:hidden">Table of Contents</summary><div class="py-2 border-dotted border-neutral-300 ltr:-ml-5 ltr:border-l ltr:pl-5 rtl:-mr-5 rtl:border-r rtl:pr-5 dark:border-neutral-600"><nav id=TableOfContents><ul><li><ul><li><a href=#tldr>TL;DR</a><ul><li><a href=#why-does-it-matter>Why does it matter?</a></li><li><a href=#key-takeaways>Key Takeaways</a></li><li><a href=#visual-insights>Visual Insights</a></li></ul></li><li><a href=#more-visual-insights>More visual insights</a></li><li><a href=#full-paper>Full paper</a></li></ul></li></ul></nav></div></details><script>var margin=200,marginError=50;(function(){var t=$(window),e=$("#TOCView"),s=e.height();function n(){var n=t.height()-margin;s>=n?(e.css("overflow-y","scroll"),e.css("max-height",n+marginError+"px")):(e.css("overflow-y","hidden"),e.css("max-height","9999999px"))}t.on("resize",n),$(document).ready(n)})()</script></div></div><div class="min-w-0 min-h-0 max-w-fit"><div class="article-content max-w-prose mb-20"><h3 class="relative group">TL;DR<div id=tldr class=anchor></div><span class="absolute top-0 w-6 transition-opacity opacity-0 ltr:-left-6 rtl:-right-6 not-prose group-hover:opacity-100"><a class="group-hover:text-primary-300 dark:group-hover:text-neutral-700" style=text-decoration-line:none!important href=#tldr aria-label=Anchor>#</a></span></h3><div class="lead text-neutral-500 dark:text-neutral-400 !mb-9 text-xl">DPLM-2 is a groundbreaking multimodal protein language model that overcomes limitations of previous approaches by jointly modeling protein sequences and 3D structures. It uses a novel lookup-free quantization method to represent structures as discrete tokens for language model processing. Trained on a combination of experimental and synthetic data, DPLM-2 achieves state-of-the-art performance in several key tasks, including unconditional protein generation (simultaneously creating sequence and structure), folding (predicting 3D structure from sequence), inverse folding (generating sequence from structure), and motif scaffolding (designing protein structures with specific functional motifs). The model&rsquo;s ability to handle these tasks effectively makes it a valuable tool for protein engineering and drug discovery. It also leverages pre-trained sequence-based models efficiently, reducing computational costs. The open-source nature of the model further enhances its impact on the research community.</div><a class="!rounded-md bg-primary-600 px-4 py-2 !text-neutral !no-underline hover:!bg-primary-500 dark:bg-primary-800 dark:hover:!bg-primary-700" href=https://arxiv.org/abs/2410.13782 target=_self role=button><span class="relative inline-block align-text-bottom icon"><svg width="800" height="800" viewBox="0 0 16 16" fill="none"><path d="M7.05025 1.53553C8.03344.552348 9.36692.0 10.7574.0 13.6528.0 16 2.34721 16 5.24264 16 6.63308 15.4477 7.96656 14.4645 8.94975L12.4142 11 11 9.58579l2.0503-2.05026C13.6584 6.92742 14 6.10264 14 5.24264 14 3.45178 12.5482 2 10.7574 2c-.86004.0-1.68482.34163-2.29293.94975L6.41421 5 5 3.58579 7.05025 1.53553z" fill="#000"/><path d="M7.53553 13.0503 9.58579 11 11 12.4142 8.94975 14.4645C7.96656 15.4477 6.63308 16 5.24264 16 2.34721 16 0 13.6528.0 10.7574.0 9.36693.552347 8.03344 1.53553 7.05025L3.58579 5 5 6.41421 2.94975 8.46447C2.34163 9.07258 2 9.89736 2 10.7574 2 12.5482 3.45178 14 5.24264 14c.86.0 1.68478-.3416 2.29289-.9497z" fill="#000"/><path d="M5.70711 11.7071 11.7071 5.70711 10.2929 4.29289 4.29289 10.2929l1.41422 1.4142z" fill="#000"/></svg>
</span>&nbsp; read the paper on arXiv</a><h4 class="relative group">Why does it matter?<div id=why-does-it-matter class=anchor></div><span class="absolute top-0 w-6 transition-opacity opacity-0 ltr:-left-6 rtl:-right-6 not-prose group-hover:opacity-100"><a class="group-hover:text-primary-300 dark:group-hover:text-neutral-700" style=text-decoration-line:none!important href=#why-does-it-matter aria-label=Anchor>#</a></span></h4><p>This paper is important because it introduces a novel multimodal protein language model, DPLM-2, advancing the field of generative protein modeling. Its ability to simultaneously generate compatible protein sequences and structures opens new avenues for protein design and engineering. The efficient use of existing resources and open-source nature promotes wider adoption and collaborative advancements.</p><h4 class="relative group">Key Takeaways<div id=key-takeaways class=anchor></div><span class="absolute top-0 w-6 transition-opacity opacity-0 ltr:-left-6 rtl:-right-6 not-prose group-hover:opacity-100"><a class="group-hover:text-primary-300 dark:group-hover:text-neutral-700" style=text-decoration-line:none!important href=#key-takeaways aria-label=Anchor>#</a></span></h4><div class="flex px-4 py-3 rounded-md bg-primary-100 dark:bg-primary-900"><span class="text-primary-400 ltr:pr-3 rtl:pl-3 flex items-center"><span class="relative block icon"><svg width="800" height="800" viewBox="0 0 24 24" fill="none"><path d="M9.15316 5.40838C10.4198 3.13613 11.0531 2 12 2s1.5802 1.13612 2.8468 3.40837l.3277.58786C15.5345 6.64193 15.7144 6.96479 15.9951 7.17781c.2806.21302.629999999999999.29209 1.329.45024L17.9605 7.77203C20.4201 8.32856 21.65 8.60682 21.9426 9.54773c.2926.94087-.5458 1.92137-2.2227 3.88217L19.2861 13.9372C18.8096 14.4944 18.5713 14.773 18.4641 15.1177 18.357 15.4624 18.393 15.8341 18.465 16.5776L18.5306 17.2544C18.7841 19.8706 18.9109 21.1787 18.1449 21.7602 17.3788 22.3417 16.2273 21.8115 13.9243 20.7512L13.3285 20.4768C12.6741 20.1755 12.3469 20.0248 12 20.0248S11.3259 20.1755 10.6715 20.4768L10.0757 20.7512c-2.30302 1.0603-3.45452 1.5905-4.22055 1.009C5.08912 21.1787 5.21588 19.8706 5.4694 17.2544L5.53498 16.5776C5.60703 15.8341 5.64305 15.4624 5.53586 15.1177 5.42868 14.773 5.19043 14.4944 4.71392 13.9372L4.2801 13.4299c-1.67685-1.9608-2.51528-2.9413-2.22268-3.88217C2.35002 8.60682 3.57986 8.32856 6.03954 7.77203l.63635-.14398C7.37485 7.4699 7.72433 7.39083 8.00494 7.17781 8.28555 6.96479 8.46553 6.64194 8.82547 5.99623L9.15316 5.40838z" fill="#1c274c"/></svg>
</span></span><span class=dark:text-neutral-300><div id=typeit-0c5623c826e786f62aac3d720c67c6c8></div><script>document.addEventListener("DOMContentLoaded",function(){new TypeIt("#typeit-0c5623c826e786f62aac3d720c67c6c8",{strings:[" DPLM-2 generates highly accurate and diverse protein sequences and 3D structures simultaneously. "],speed:10,lifeLike:!0,startDelay:0,breakLines:!0,waitUntilVisible:!0,loop:!1}).go()})</script></span></div><div class="flex px-4 py-3 rounded-md bg-primary-100 dark:bg-primary-900"><span class="text-primary-400 ltr:pr-3 rtl:pl-3 flex items-center"><span class="relative block icon"><svg width="800" height="800" viewBox="0 0 24 24" fill="none"><path d="M9.15316 5.40838C10.4198 3.13613 11.0531 2 12 2s1.5802 1.13612 2.8468 3.40837l.3277.58786C15.5345 6.64193 15.7144 6.96479 15.9951 7.17781c.2806.21302.629999999999999.29209 1.329.45024L17.9605 7.77203C20.4201 8.32856 21.65 8.60682 21.9426 9.54773c.2926.94087-.5458 1.92137-2.2227 3.88217L19.2861 13.9372C18.8096 14.4944 18.5713 14.773 18.4641 15.1177 18.357 15.4624 18.393 15.8341 18.465 16.5776L18.5306 17.2544C18.7841 19.8706 18.9109 21.1787 18.1449 21.7602 17.3788 22.3417 16.2273 21.8115 13.9243 20.7512L13.3285 20.4768C12.6741 20.1755 12.3469 20.0248 12 20.0248S11.3259 20.1755 10.6715 20.4768L10.0757 20.7512c-2.30302 1.0603-3.45452 1.5905-4.22055 1.009C5.08912 21.1787 5.21588 19.8706 5.4694 17.2544L5.53498 16.5776C5.60703 15.8341 5.64305 15.4624 5.53586 15.1177 5.42868 14.773 5.19043 14.4944 4.71392 13.9372L4.2801 13.4299c-1.67685-1.9608-2.51528-2.9413-2.22268-3.88217C2.35002 8.60682 3.57986 8.32856 6.03954 7.77203l.63635-.14398C7.37485 7.4699 7.72433 7.39083 8.00494 7.17781 8.28555 6.96479 8.46553 6.64194 8.82547 5.99623L9.15316 5.40838z" fill="#1c274c"/></svg>
</span></span><span class=dark:text-neutral-300><div id=typeit-21b8654b5b053fec22da0b52bb8184b9></div><script>document.addEventListener("DOMContentLoaded",function(){new TypeIt("#typeit-21b8654b5b053fec22da0b52bb8184b9",{strings:[" The model excels in various protein generation tasks, including folding, inverse folding, and motif scaffolding. "],speed:10,lifeLike:!0,startDelay:1e3,breakLines:!0,waitUntilVisible:!0,loop:!1}).go()})</script></span></div><div class="flex px-4 py-3 rounded-md bg-primary-100 dark:bg-primary-900"><span class="text-primary-400 ltr:pr-3 rtl:pl-3 flex items-center"><span class="relative block icon"><svg width="800" height="800" viewBox="0 0 24 24" fill="none"><path d="M9.15316 5.40838C10.4198 3.13613 11.0531 2 12 2s1.5802 1.13612 2.8468 3.40837l.3277.58786C15.5345 6.64193 15.7144 6.96479 15.9951 7.17781c.2806.21302.629999999999999.29209 1.329.45024L17.9605 7.77203C20.4201 8.32856 21.65 8.60682 21.9426 9.54773c.2926.94087-.5458 1.92137-2.2227 3.88217L19.2861 13.9372C18.8096 14.4944 18.5713 14.773 18.4641 15.1177 18.357 15.4624 18.393 15.8341 18.465 16.5776L18.5306 17.2544C18.7841 19.8706 18.9109 21.1787 18.1449 21.7602 17.3788 22.3417 16.2273 21.8115 13.9243 20.7512L13.3285 20.4768C12.6741 20.1755 12.3469 20.0248 12 20.0248S11.3259 20.1755 10.6715 20.4768L10.0757 20.7512c-2.30302 1.0603-3.45452 1.5905-4.22055 1.009C5.08912 21.1787 5.21588 19.8706 5.4694 17.2544L5.53498 16.5776C5.60703 15.8341 5.64305 15.4624 5.53586 15.1177 5.42868 14.773 5.19043 14.4944 4.71392 13.9372L4.2801 13.4299c-1.67685-1.9608-2.51528-2.9413-2.22268-3.88217C2.35002 8.60682 3.57986 8.32856 6.03954 7.77203l.63635-.14398C7.37485 7.4699 7.72433 7.39083 8.00494 7.17781 8.28555 6.96479 8.46553 6.64194 8.82547 5.99623L9.15316 5.40838z" fill="#1c274c"/></svg>
</span></span><span class=dark:text-neutral-300><div id=typeit-13e25f5b0aa9cad10aea27422468cac0></div><script>document.addEventListener("DOMContentLoaded",function(){new TypeIt("#typeit-13e25f5b0aa9cad10aea27422468cac0",{strings:[" DPLM-2's structure-aware representations improve the performance of various protein predictive tasks. "],speed:10,lifeLike:!0,startDelay:2e3,breakLines:!0,waitUntilVisible:!0,loop:!1}).go()})</script></span></div><hr><h4 class="relative group">Visual Insights<div id=visual-insights class=anchor></div><span class="absolute top-0 w-6 transition-opacity opacity-0 ltr:-left-6 rtl:-right-6 not-prose group-hover:opacity-100"><a class="group-hover:text-primary-300 dark:group-hover:text-neutral-700" style=text-decoration-line:none!important href=#visual-insights aria-label=Anchor>#</a></span></h4><p><figure><img class="my-0 rounded-md" loading=lazy srcset="/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_2_0_hu1838437032599710792.png 330w,
/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_2_0_hu2821497031384567480.png 660w,
/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_2_0_hu6154819568707303956.png 1024w,
/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_2_0_hu5895381254206560536.png 2x" src=/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_2_0_hu2821497031384567480.png alt><figcaption>ğŸ”¼ Figure 1: Overall illustration of DPLM-2. (A) Structure tokenization consists of a GVP-based encoder to yield invariant backbone geometric features, a lookup-free quantizer (LFQ) to discretize encoded structural features into structure tokens within a codebook, and an IPA-based decoder as de-tokenizer to convert structure tokens back to backbone atomic coordinates. (B) Multimodal learning and generation of protein structure and sequence with DPLM-2. (C) Various applications of DPLM-2 as a protein foundation model: (1) unconditional protein sequence-structure mixed-modal co-generation; (2) protein sequence-structure joint representation for predictive tasks; (3) structure prediction; (4) fixed-backbone sequence generation; (5) conditional protein generation with structure-sequence mixed-modal input and output.</figcaption></figure></p><blockquote><p>Figure 1 illustrates the overall architecture of DPLM-2, detailing its structure tokenization, multimodal learning and generation process, and various applications as a protein foundation model.</p></blockquote><p><figure><img class="my-0 rounded-md" loading=lazy srcset="/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_0_hu9312768046284608241.png 330w,
/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_0_hu12021902137033743672.png 660w,
/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_0_hu775100049426905965.png 1024w,
/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_0_hu9832964978549206651.png 2x" src=/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_0_hu12021902137033743672.png alt><figcaption>ğŸ”¼ Figure 3: Evaluation of DPLM-2 on unconditional structure-sequence co-generation. Here for designability of co-generated proteins, we use ESMFold to obtain refolded structure of DPLM-2-generated sequence and measure the structural similarity between DPLM-2-generated structure and the refolded structure, which aims to measure the compatibility of the co-generated structure and sequence pairs.</figcaption></figure></p><blockquote><p>Figure 3 shows the evaluation results of DPLM-2 on unconditional protein generation, illustrating the model&rsquo;s ability to generate diverse, high-quality proteins with compatible structure and sequence.</p></blockquote><figure style=max-width:100%;text-align:center><table style=width:100%><caption style=caption-side:bottom;text-align:left;white-space:normal;display:block;max-width:100%;color:var(--tw-prose-captions);margin-bottom:10px>ğŸ”½ Table 2: Benchmarking comparison of unconditional protein generation, in terms of structure-sequence co-generation, backbone-only generation, and sequence-only generation. For each method, we generate 100 samples for lengths in [100, 200, 300, 400, 500]. * denotes Multiflow variants retrained by us using different dataset â€“ native PDB data without ProteinMPNN distillation and the same training data as DPLM-2 (i.e., PDB+SwissProt), respectively.</caption><table id=3 style=font-size:14px><tr><td></td><td colspan=3>Quality</td><td rowspan=2>Novelty avg. pdb-TM (â†“)</td><td colspan=2>Diversity</td></tr><tr><td></td><td>scTM (â†‘)</td><td>scRMSD (â†“)</td><td>pLDDT (â†‘)</td><td>avg. inner-TM (â†“)</td><td>MaxCluster (â†‘)</td></tr><tr><td>Structure-sequence co-generation.</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Native PDB protein</td><td>4.623 å£« 5.688</td><td>0.904 åœŸ 0.129</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>ESM3-Open (1.4B, seq â†’ struct)</td><td>0.624 å£« 0.232</td><td>24.180 åœŸ 24.109</td><td>-</td><td>0.660 åœŸ 0.000</td><td>0.410 åœŸ 0.167</td><td>0.540</td></tr><tr><td>MultiFlow w/ distillation (official ckpt)</td><td>0.930 åœŸ 0.098</td><td>3.208 åœŸ 4,741</td><td>79.447</td><td>0.704 å£« 0.000</td><td>0.468 åœŸ 0.152</td><td>0.500</td></tr><tr><td>*MultiFlow w/o distillation</td><td>0.750 å£« 0.163</td><td>9.306 åœŸ 8.499</td><td>65.861</td><td></td><td></td><td></td></tr><tr><td>*MultiFlow (retrained on our training data)</td><td>0.871 åœŸ 0.934</td><td>6.580 åœŸ 6.258</td><td>62.624</td><td></td><td></td><td></td></tr><tr><td>DPLM-2 (650M, seq â†’ struct)</td><td>0.907 å£« 0.117</td><td>6.337 å£« 9.403</td><td>82.246</td><td>0.653 åœŸ 0.195</td><td>0.594 å£« 0.270</td><td>0.651</td></tr><tr><td>DPLM-2 (650M, struct â†’ seq)</td><td>0.921 å£« 0.098</td><td>4.969 å£« 6.735</td><td>81.910</td><td>0.637 å£« 0.195</td><td>0.679 å£« 0.288</td><td>0.575</td></tr><tr><td>DPLM-2 (650M, co-generation)</td><td>0.925 å£« 0.085</td><td>3.899 å£« 3.723</td><td>82.686</td><td>0.640 åœŸ 0.204</td><td>0.703 å£« 0.279</td><td>0.545</td></tr><tr><td colspan=7>Unconditional backbone generation. (sequence predicted by ProteinMPNN)</td></tr><tr><td>Native PDB struct. (seq. from PMPNN)</td><td>0.969 å£« 0.000</td><td>0.864 åœŸ 0.000</td><td></td><td>-</td><td>0.282 å£« 0.000</td><td>0.782</td></tr><tr><td>FrameDiff</td><td>0.818 å£« 0.000</td><td>3.919 åœŸ 0.000</td><td></td><td>0.668 åœŸ 0.000</td><td>0.465 å£« 0.000</td><td>0.252</td></tr><tr><td>FoldFlow</td><td>0.540 å£« 0.000</td><td>7.965 å£« 0.000</td><td>-</td><td>0.566 å£« 0.000</td><td>0.411 å£« 0.000</td><td>0.762</td></tr><tr><td>RFDiffusion</td><td>0.914 åœŸ 0.000</td><td>1.969 åœŸ 0.000</td><td>-</td><td>0.657 å£« 0.000</td><td>0.363 åœŸ 0.000</td><td>0.598</td></tr><tr><td>DPLM-2 (650M)</td><td>0.945 åœŸ 0.082</td><td>4.451 å£« 5.261</td><td>-</td><td>0.637 å£« 0.195</td><td>0.679 å£« 0.288</td><td>0.575</td></tr><tr><td colspan=7>Unconditional sequence generation. (structures predicted by ESMFold)</td></tr><tr><td>EvoDiff</td><td></td><td></td><td>35.846</td><td>0.432 å£« 0.106</td><td>0.366 å£« 0.070</td><td>0.990</td></tr><tr><td>DPLM (650M)</td><td></td><td></td><td>83.252</td><td>0.541 åœŸ 0.187</td><td>0.515 åœŸ 0.222</td><td>0.735</td></tr><tr><td>DPLM-2 (650M)</td><td></td><td></td><td>82.246</td><td>0.662 å£« 0.199</td><td>0.589 å£« 0.268</td><td>0.700</td></tr><tr><td colspan=7>A - stats of secondary structure B - impact of secondary structure on designability 1.2 Helix 12 14 20 Sheet 1.0 Loop 10 12 0.8 15 10 8 Proportion 0.4 bb 0.6 rmsd 8 rmsd 6 rmsd 10 bb 6 bb 4 0.2 4 5 2 0.0 2 T 1 å·¥ T T 0 -0.2 0 0 RFDiffusion MultiFlow ESM3 PDB DPLM2 0.6 0.7 0.0 0.1 0.2 0.3 0.4 0.5 0.60.7 0.8 0.0 0.1 0.2 0.3 0.4 0.5 0.6 0.1 0.2 0.3 0.4 0.5 helix ratio strand ratio loop ratio C - unconditionally-generated proteins from different models 0010 0.2 0.8 20 0 0.6 ã‚‹ CRM5D Loo MultiFlow + 0.4 0.8 0.2 5 1/g 0.0 0.0 0.2 0.4 0.6 0.8 1/0 helix 0.0 1.0 25 PDB ì„± scRMSD 0.2 5 0.0 0.8</td></tr><tr><td></td><td></td><td></td><td>1/0 25 è€Œ</td><td>RâŒ€</td><td>0.8 0.6 ã‚‹</td><td>å±‚</td></tr><tr><td>DPLM2</td><td>1 0.0 0.2 0.4 0</td><td>1.0 0.8</td><td>ã®</td><td>0.0</td><td></td><td></td></tr><tr><td></td><td>Length:</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>Length: 100</td><td>300</td><td></td><td></td><td></td><td>0.2 5</td></tr><tr><td>0.6 å°” scRMSD Length: Figure 4: Analysis regarding secondary structure of generated proteins. (A) Statistics of averaged</td><td>70</td><td>200 Length:</td><td>ë£Œ Length: 400</td><td>Length: 500</td><td>0.0 0.2 0.4 helix</td><td>0.6 0.8 1/0 0.0</td></tr></table></table></figure><blockquote><p>Table 2 compares the performance of DPLM-2 against other unconditional protein generation methods across various metrics, including quality, novelty, and diversity of generated sequences and structures.</p></blockquote><h3 class="relative group">More visual insights<div id=more-visual-insights class=anchor></div><span class="absolute top-0 w-6 transition-opacity opacity-0 ltr:-left-6 rtl:-right-6 not-prose group-hover:opacity-100"><a class="group-hover:text-primary-300 dark:group-hover:text-neutral-700" style=text-decoration-line:none!important href=#more-visual-insights aria-label=Anchor>#</a></span></h3><details><summary>More on figures</summary><p><figure><img class="my-0 rounded-md" loading=lazy srcset="/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_4_0_hu6953129119166338877.png 330w,
/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_4_0_hu3346082585083048478.png 660w,
/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_4_0_hu6258055609300715326.png 1024w,
/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_4_0_hu1085119888066626632.png 2x" src=/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_4_0_hu3346082585083048478.png alt><figcaption>ğŸ”¼ Figure 1: Overall illustration of DPLM-2. (A) Structure tokenization consists of a GVP-based encoder to yield invariant backbone geometric features, a lookup-free quantizer (LFQ) to discretize encoded structural features into structure tokens within a codebook, and an IPA-based decoder as de-tokenizer to convert structure tokens back to backbone atomic coordinates. (B) Multimodal learning and generation of protein structure and sequence with DPLM-2. (C) Various applications of DPLM-2 as a protein foundation model: (1) unconditional protein sequence-structure mixed-modal co-generation; (2) protein sequence-structure joint representation for predictive tasks; (3) structure prediction; (4) fixed-backbone sequence generation; (5) conditional protein generation with structure-sequence mixed-modal input and output.</figcaption></figure></p><blockquote><p>Figure 1 illustrates the overall architecture of DPLM-2, including structure tokenization, multimodal training and sampling process, and various downstream applications.</p></blockquote><p><figure><img class="my-0 rounded-md" loading=lazy srcset="/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_7_0_hu4335745427998897851.png 330w,
/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_7_0_hu13278000517041589482.png 660w,
/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_7_0_hu2574652150883936870.png 1024w,
/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_7_0_hu3478802609261112070.png 2x" src=/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_7_0_hu13278000517041589482.png alt><figcaption>ğŸ”¼ Figure 1: Overall illustration of DPLM-2. (A) Structure tokenization consists of a GVP-based encoder to yield invariant backbone geometric features, a lookup-free quantizer (LFQ) to discretize encoded structural features into structure tokens within a codebook, and an IPA-based decoder as de-tokenizer to convert structure tokens back to backbone atomic coordinates. (B) Multimodal learning and generation of protein structure and sequence with DPLM-2. (C) Various applications of DPLM-2 as a protein foundation model: (1) unconditional protein sequence-structure mixed-modal co-generation; (2) protein sequence-structure joint representation for predictive tasks; (3) structure prediction; (4) fixed-backbone sequence generation; (5) conditional protein generation with structure-sequence mixed-modal input and output.</figcaption></figure></p><blockquote><p>Figure 1 provides a comprehensive overview of the DPLM-2 model, illustrating its structure tokenization process, multimodal learning and generation capabilities, and various applications in protein modeling tasks.</p></blockquote><p><figure><img class="my-0 rounded-md" loading=lazy srcset="/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_7_1_hu5785398310472702788.png 330w,
/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_7_1_hu16909683182274549869.png 660w,
/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_7_1_hu4936928481033136361.png 1024w,
/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_7_1_hu15158938798895054811.png 2x" src=/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_7_1_hu16909683182274549869.png alt><figcaption>ğŸ”¼ Figure 1: Overall illustration of DPLM-2. (A) Structure tokenization consists of a GVP-based encoder to yield invariant backbone geometric features, a lookup-free quantizer (LFQ) to discretize encoded structural features into structure tokens within a codebook, and an IPA-based decoder as de-tokenizer to convert structure tokens back to backbone atomic coordinates. (B) Multimodal learning and generation of protein structure and sequence with DPLM-2. (C) Various applications of DPLM-2 as a protein foundation model: (1) unconditional protein sequence-structure mixed-modal co-generation; (2) protein sequence-structure joint representation for predictive tasks; (3) structure prediction; (4) fixed-backbone sequence generation; (5) conditional protein generation with structure-sequence mixed-modal input and output.</figcaption></figure></p><blockquote><p>Figure 1 illustrates the overall architecture of DPLM-2, including structure tokenization, multimodal training and sampling, and various applications as a protein foundation model.</p></blockquote><p><figure><img class="my-0 rounded-md" loading=lazy srcset="/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_7_2_hu4594078215994323915.png 330w,
/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_7_2_hu1179421559073287700.png 660w,
/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_7_2_hu15259219401287870945.png 1024w,
/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_7_2_hu8407892535548230069.png 2x" src=/ai-paper-reviewer/paper-reviews/2410.13782/figures/figures_7_2_hu1179421559073287700.png alt><figcaption>ğŸ”¼ Figure 3: Evaluation of DPLM-2 on unconditional structure-sequence co-generation. Here for designability of co-generated proteins, we use ESMFold to obtain refolded structure of DPLM-2-generated sequence and measure the structural similarity between DPLM-2-generated structure and the refolded structure, which aims to measure the compatibility of the co-generated structure and sequence pairs.</figcaption></figure></p><blockquote><p>Figure 3 presents the results of DPLM-2 for unconditional protein generation, showing its ability to generate diverse and high-quality proteins with simultaneous structure-sequence co-generation, as well as its performance in structure novelty, diversity, and length extrapolation.</p></blockquote></details><details><summary>More on charts</summary><p><figure><img class="my-0 rounded-md" loading=lazy srcset="/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_1_hu16547887734716410084.png 330w,
/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_1_hu15189249403699998465.png 660w,
/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_1_hu14151719074839336546.png 1024w,
/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_1_hu6205973803860761974.png 2x" src=/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_1_hu15189249403699998465.png alt><figcaption>ğŸ”¼ Figure 3: Evaluation of DPLM-2 on unconditional structure-sequence co-generation. Here for designability of co-generated proteins, we use ESMFold to obtain refolded structure of DPLM-2-generated sequence and measure the structural similarity between DPLM-2-generated structure and the refolded structure, which aims to measure the compatibility of the co-generated structure and sequence pairs.</figcaption></figure></p><blockquote><p>Figure 3 presents the evaluation results of DPLM-2&rsquo;s unconditional protein generation, showcasing its ability to generate diverse and high-quality proteins with simultaneous structure and sequence co-generation, demonstrating protein designability, structure diversity, sequence foldability, and structure novelty across various protein lengths.</p></blockquote><p><figure><img class="my-0 rounded-md" loading=lazy srcset="/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_2_hu7730440599504127334.png 330w,
/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_2_hu6378779052032078224.png 660w,
/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_2_hu527536782421135106.png 1024w,
/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_2_hu12465205663421196391.png 2x" src=/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_2_hu6378779052032078224.png alt><figcaption>ğŸ”¼ Figure 3: Evaluation of DPLM-2 on unconditional structure-sequence co-generation. Here for designability of co-generated proteins, we use ESMFold to obtain refolded structure of DPLM-2-generated sequence and measure the structural similarity between DPLM-2-generated structure and the refolded structure, which aims to measure the compatibility of the co-generated structure and sequence pairs.</figcaption></figure></p><blockquote><p>Figure 3 presents the evaluation results of DPLM-2 for unconditional protein generation, showing the designability, diversity, novelty, and length extrapolation of the generated protein sequences and structures.</p></blockquote><p><figure><img class="my-0 rounded-md" loading=lazy srcset="/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_3_hu18017417718050626151.png 330w,
/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_3_hu1720002205871725506.png 660w,
/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_3_hu9704707498236363199.png 1024w,
/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_3_hu3545650945116373763.png 2x" src=/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_3_hu1720002205871725506.png alt><figcaption>ğŸ”¼ Figure 3: Evaluation of DPLM-2 on unconditional structure-sequence co-generation. Here for designability of co-generated proteins, we use ESMFold to obtain refolded structure of DPLM-2-generated sequence and measure the structural similarity between DPLM-2-generated structure and the refolded structure, which aims to measure the compatibility of the co-generated structure and sequence pairs.</figcaption></figure></p><blockquote><p>Figure 3 shows the evaluation results of DPLM-2 on unconditional protein generation, assessing designability, diversity, novelty, and length extrapolation of generated protein sequences and structures.</p></blockquote><p><figure><img class="my-0 rounded-md" loading=lazy srcset="/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_4_hu3001296838116818150.png 330w,
/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_4_hu17377525273422554761.png 660w,
/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_4_hu11544213548779644054.png 1024w,
/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_4_hu3396422997903844300.png 2x" src=/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_7_4_hu17377525273422554761.png alt><figcaption>ğŸ”¼ Figure 3: Evaluation of DPLM-2 on unconditional structure-sequence co-generation. Here for designability of co-generated proteins, we use ESMFold to obtain refolded structure of DPLM-2-generated sequence and measure the structural similarity between DPLM-2-generated structure and the refolded structure, which aims to measure the compatibility of the co-generated structure and sequence pairs.</figcaption></figure></p><blockquote><p>Figure 3 presents the evaluation results of DPLM-2&rsquo;s unconditional structure-sequence co-generation performance in terms of designability, diversity, novelty, and length extrapolation.</p></blockquote><p><figure><img class="my-0 rounded-md" loading=lazy srcset="/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_10_0_hu12189436185582665744.png 330w,
/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_10_0_hu10143419913908457486.png 660w,
/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_10_0_hu18243263701148964570.png 1024w,
/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_10_0_hu8155635065321861420.png 2x" src=/ai-paper-reviewer/paper-reviews/2410.13782/charts/charts_10_0_hu10143419913908457486.png alt><figcaption>ğŸ”¼ Figure 5: Evaluation of motif-scaffolding w.r.t. success rate and num. of solved problems.</figcaption></figure></p><blockquote><p>The chart compares the performance of different models (EvoDiff, DPLM, ESM3, DPLM2, RFDiffusion) on motif-scaffolding tasks, showing the number of solved problems and success rates for sequence-based, structure-based, and co-generation approaches.</p></blockquote></details><details><summary>More on tables</summary><figure style=max-width:100%;text-align:center><table style=width:100%><caption style=caption-side:bottom;text-align:left;white-space:normal;display:block;max-width:100%;color:var(--tw-prose-captions);margin-bottom:10px>ğŸ”½ Table 3: Ablation study on the sequence pre-training and training data augmentation.</caption><br><table id=5 style=font-size:14px><tr><td rowspan=2>sequence pre-training</td><td rowspan=2>synthetic structures</td><td colspan=2>length 100</td><td colspan=2>length 200</td><td colspan=2>length 300</td><td colspan=2>length 400</td><td colspan=2>length 500</td></tr><tr><td>scTM</td><td>clusters</td><td>scTM</td><td>clusters</td><td>scTM</td><td>clusters</td><td>scTM</td><td>clusters</td><td>scTM</td><td>clusters</td></tr><tr><td>X</td><td>X</td><td>0.9241</td><td>20</td><td>0.8674</td><td>34</td><td>0.7667</td><td>33</td><td>0.5016</td><td>25</td><td>0.4511</td><td>25</td></tr><tr><td>V</td><td>X</td><td>0.9610</td><td>26</td><td>0.9349</td><td>47</td><td>0.9169</td><td>38</td><td>0.8643</td><td>35</td><td>0.7673</td><td>52</td></tr><tr><td>X</td><td>V</td><td>0.8988</td><td>27</td><td>0.9182</td><td>15</td><td>0.9343</td><td>13</td><td>0.8518</td><td>21</td><td>0.8288</td><td>31</td></tr><tr><td></td><td>V</td><td>0.9348</td><td>35</td><td>0.9428</td><td>40</td><td>0.9232</td><td>48</td><td>0.9260</td><td>40</td><td>0.9012</td><td>32</td></tr></table></table></figure><blockquote><p>Table 3 shows the ablation study results on the effects of sequence pre-training and data augmentation on the unconditional protein generation performance of DPLM-2, demonstrating that both strategies significantly improve the model&rsquo;s performance, especially in generating long proteins.</p></blockquote><figure style=max-width:100%;text-align:center><table style=width:100%><caption style=caption-side:bottom;text-align:left;white-space:normal;display:block;max-width:100%;color:var(--tw-prose-captions);margin-bottom:10px>ğŸ”½ Table 4: Structure prediction performance comparison between DPLM-2 and different baseline approaches on CAMEO 2022 datasets. â€ : PVQD results are quoted from Liu et al. (2023).</caption><table id=11 style=font-size:14px><tr><td rowspan=2>Models</td><td colspan=2>CAMEO 2022</td><td colspan=2>PDB date split</td></tr><tr><td>RMSD</td><td>TMscore</td><td>RMSD</td><td>TMscore</td></tr><tr><td>ESMFold</td><td>3.99/2.03</td><td>0.85/0.93</td><td>2.84/1.19</td><td>0.93/0.97</td></tr><tr><td>+PVQD</td><td>4.08/1.95</td><td>0.81/0.88</td><td>-</td><td>-</td></tr><tr><td>MultiFlow</td><td>17.84/17.96</td><td>0.50/0.46</td><td>15.64/16.08</td><td>0.53/0.49</td></tr><tr><td>ESM3</td><td>6.33/2.98</td><td>0.85/0.92</td><td>4.94/2.28</td><td>0.87/0.93</td></tr><tr><td>DPLM-2 (150M)</td><td>9.22/7.64</td><td>0.75/0.81</td><td>8.35/5.60</td><td>0.76/0.82</td></tr><tr><td>w/ folding SFT</td><td>7.66/4.37</td><td>0.80/0.86</td><td>6.00/3.41</td><td>0.83/0.88</td></tr><tr><td>DPLM-2 (650M)</td><td>7.37/4.89</td><td>0.79/0.86</td><td>5.67/3.33</td><td>0.83/0.88</td></tr><tr><td>w/ folding SFT</td><td>6.21/3.78</td><td>0.84/0.89</td><td>3.40/1.78</td><td>0.89/0.94</td></tr><tr><td>DPLM-2 (3B)</td><td>6.34/3.65</td><td>0.83/0.89</td><td>4.54/2.54</td><td>0.86/0.92</td></tr><tr><td>w/ folding SFT</td><td>5.71/3.23</td><td>0.85/0.90</td><td>3.15/1.69</td><td>0.90/0.95</td></tr></table></table></figure><blockquote><p>Table 4 compares the structure prediction performance of DPLM-2 against several other state-of-the-art methods on the CAMEO 2022 dataset, using RMSD and TMscore as evaluation metrics.</p></blockquote><figure style=max-width:100%;text-align:center><table style=width:100%><caption style=caption-side:bottom;text-align:left;white-space:normal;display:block;max-width:100%;color:var(--tw-prose-captions);margin-bottom:10px>ğŸ”½ Table 5: Comparison on inverse folding task.</caption><br><table id=6 style=font-size:16px><tr><td rowspan=2>Models</td><td colspan=2>CAMEO 2022</td><td colspan=2>PDB date split</td></tr><tr><td>AAR</td><td>scTM</td><td>AAR</td><td>scTM</td></tr><tr><td>MultiFlow</td><td>32.28/33.58</td><td>0.87/0.94</td><td>37.74/37.59</td><td>0.94/0.96</td></tr><tr><td>ESM3</td><td>47.06/46.24</td><td>0.90/0.95</td><td>49.50/49.42</td><td>0.94/0.97</td></tr><tr><td>DPLM-2 (150M)</td><td>45.22/46.12</td><td>0.87/0.93</td><td>48.83/47.96</td><td>0.89/0.95</td></tr><tr><td>DPLM-2 (650M)</td><td>49.01/50.10</td><td>0.88/0.93</td><td>54.80/53/07</td><td>0.91/0.96</td></tr><tr><td>DPLM-2 (3B)</td><td>52.36/53.72</td><td>0.89/0.95</td><td>61.67/57.91</td><td>0.92/0.96</td></tr></table></table></figure><blockquote><p>Table 5 presents the performance comparison of different models on the inverse folding task, evaluating amino acid recovery (AAR) and structure consistency (scTM).</p></blockquote><figure style=max-width:100%;text-align:center><table style=width:100%><caption style=caption-side:bottom;text-align:left;white-space:normal;display:block;max-width:100%;color:var(--tw-prose-captions);margin-bottom:10px>ğŸ”½ Table 6: Performance on various protein predictive downstream tasks. â€ : benchmarked results are quoted from Su et al. (2023).</caption><br><table id=14 style=font-size:14px><tr><td rowspan=3>Models</td><td rowspan=2>Thermostability</td><td rowspan=2>HumanPPI</td><td rowspan=2>Metal Ion Binding</td><td rowspan=2>EC</td><td colspan=3>GO</td><td colspan=2>DeepLoc</td></tr><tr><td>MF</td><td>BP</td><td>CC</td><td>Subcellular</td><td>Binary</td></tr><tr><td>Spearman&rsquo; S P</td><td>Acc (%)</td><td>Acc (%)</td><td>Fmax</td><td>Fmax</td><td>Fmax</td><td>Fmax</td><td>Acc (%)</td><td>Acc (%)</td></tr><tr><td>â€ SaProt (650M)</td><td>0.724</td><td>86.41</td><td>75.75</td><td>0.884</td><td>0.678</td><td>0.356</td><td>0.414</td><td>85.57</td><td>93.55</td></tr><tr><td>+MIF-ST (Yang et al., 2022b)</td><td>0.694</td><td>75.54</td><td>75.08</td><td>0.803</td><td>0.627</td><td>0.239</td><td>0.248</td><td>78.96</td><td>91.76</td></tr><tr><td>ESM2 (650M)</td><td>0.691</td><td>84.78</td><td>71.88</td><td>0.866</td><td>0.676</td><td>0.344</td><td>0.402</td><td>83.68</td><td>92.28</td></tr><tr><td>DPLM (650M)</td><td>0.695</td><td>86.41</td><td>75.15</td><td>0.875</td><td>0.680</td><td>0.357</td><td>0.409</td><td>84.56</td><td>93.09</td></tr><tr><td>DPLM-2 (650M)</td><td>0.714</td><td>84.44</td><td>74.28</td><td>0.878</td><td>0.680</td><td>0.359</td><td>0.411</td><td>82.98</td><td>93.64</td></tr></table></table></figure><blockquote><p>Table 6 presents the performance comparison of different protein language models on various protein predictive downstream tasks, including thermostability, HumanPPI, metal ion binding, EC, GO (MF, BP, CC), DeepLoc (subcellular and binary).</p></blockquote><figure style=max-width:100%;text-align:center><table style=width:100%><caption style=caption-side:bottom;text-align:left;white-space:normal;display:block;max-width:100%;color:var(--tw-prose-captions);margin-bottom:10px>ğŸ”½ Table 8: Ablation study on the self-mixup training strategy.</caption><br><table id=7 style=font-size:14px><tr><td rowspan=2>Mixup strategy</td><td colspan=2>length 100</td><td colspan=2>length 200</td><td colspan=2>length 300</td><td colspan=2>length 400</td><td colspan=2>length 500</td></tr><tr><td>scTM</td><td>clusters</td><td>scTM</td><td>clusters</td><td>scTM</td><td>clusters</td><td>scTM</td><td>clusters</td><td>scTM</td><td>clusters</td></tr><tr><td>X</td><td>0.9237</td><td>44</td><td>0.9180</td><td>53</td><td>0.9147</td><td>48</td><td>0.9059</td><td>42</td><td>0.8896</td><td>33</td></tr><tr><td>V</td><td>0.8812</td><td>62</td><td>0.8820</td><td>62</td><td>0.9172</td><td>59</td><td>0.9099</td><td>54</td><td>0.8845</td><td>38</td></tr></table></table></figure><blockquote><p>Table 8 shows the ablation study results on the self-mixup training strategy, demonstrating its effect on the diversity of generated protein samples at different lengths.</p></blockquote><figure style=max-width:100%;text-align:center><table style=width:100%><caption style=caption-side:bottom;text-align:left;white-space:normal;display:block;max-width:100%;color:var(--tw-prose-captions);margin-bottom:10px>ğŸ”½ Table 2: Benchmarking comparison of unconditional protein generation, in terms of structure-sequence co-generation, backbone-only generation, and sequence-only generation. For each method, we generate 100 samples for lengths in [100, 200, 300, 400, 500]. * denotes Multiflow variants retrained by us using different dataset â€“ native PDB data without ProteinMPNN distillation and the same training data as DPLM-2 (i.e., PDB+SwissProt), respectively.</caption><br><table id=3 style=font-size:14px><tr><td>prediction motif-preserving designability</td><td>seqpred: V structpred: x RMSD (ESMFold ( seqpred) [motif] , structnative [motif] )&lt;1.0 pLDDT (ESMFold( seqpred) )>70</td></tr><tr><td>structure-based</td><td>seqpred: x structpred: V</td></tr><tr><td>prediction motif-preserving designability</td><td>RMSD ( ESMFold ( PMPNN ( structpred) ) [motif] , structnative [motif] )&lt;1.0 TMScore (ESMFold (PMPNN ( structpred) ) , structpred)>0.8</td></tr><tr><td>co-generation prediction motif-preserving designability</td><td>seqpredâŒ€ V structpred: V RMSD (ESMFold ( seqpred) [motif] , structnative [motif] )&lt;1.0 TMScore (ESMFold( seqpred) , structpred)>0.8</td></tr></table></table></figure><blockquote><p>Table 2 compares the performance of DPLM-2 with other unconditional protein generation methods in terms of quality, novelty, and diversity of generated protein sequences and structures.</p></blockquote><figure style=max-width:100%;text-align:center><table style=width:100%><caption style=caption-side:bottom;text-align:left;white-space:normal;display:block;max-width:100%;color:var(--tw-prose-captions);margin-bottom:10px>ğŸ”½ Table 2: Benchmarking comparison of unconditional protein generation, in terms of structure-sequence co-generation, backbone-only generation, and sequence-only generation. For each method, we generate 100 samples for lengths in [100, 200, 300, 400, 500]. * denotes Multiflow variants retrained by us using different dataset â€“ native PDB data without ProteinMPNN distillation and the same training data as DPLM-2 (i.e., PDB+SwissProt), respectively.</caption><br><table id=2 style=font-size:14px><tr><td rowspan=2></td><td colspan=4>sequence-based</td><td colspan=2>structure-based</td><td colspan=3>co-generation</td></tr><tr><td>EvoDiff</td><td>DPLM</td><td>ESM3</td><td>DPLM2</td><td>*RFDiffusion</td><td>*DPLM2</td><td>ESM3</td><td>DPLM2</td><td>*DPLM2</td></tr><tr><td>1BCF</td><td>0.00</td><td>0.00</td><td>0.89</td><td>0.01</td><td>1.00</td><td>0.07</td><td>0.23</td><td>0.01</td><td>0.05</td></tr><tr><td>1PRW</td><td>0.61</td><td>0.83</td><td>0.96</td><td>0.86</td><td>0.08</td><td>0.96</td><td>0.54</td><td>0.84</td><td>0.95</td></tr><tr><td>1QJG</td><td>0.00</td><td>0.00</td><td>0.02</td><td>0.03</td><td>0.00</td><td>0.00</td><td>0.03</td><td>0.02</td><td>0.05</td></tr><tr><td>1YCR</td><td>0.02</td><td>0.38</td><td>0.41</td><td>0.77</td><td>0.74</td><td>0.93</td><td>0.18</td><td>0.53</td><td>0.98</td></tr><tr><td>2KL8</td><td>0.04</td><td>0.08</td><td>0.11</td><td>0.47</td><td>0.88</td><td>0.94</td><td>0.11</td><td>0.57</td><td>1.00</td></tr><tr><td>3IXT</td><td>0.06</td><td>0.17</td><td>0.18</td><td>0.67</td><td>0.25</td><td>0.77</td><td>0.02</td><td>0.41</td><td>0.73</td></tr><tr><td>4JHW</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td></tr><tr><td>4ZYP</td><td>0.00</td><td>0.00</td><td>0.03</td><td>0.16</td><td>0.40</td><td>0.51</td><td>0.08</td><td>0.10</td><td>0.64</td></tr><tr><td>5IUS</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.02</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td></tr><tr><td>5TPN</td><td>0.00</td><td>0.00</td><td>0.03</td><td>0.00</td><td>0.61</td><td>0.06</td><td>0.01</td><td>0.00</td><td>0.00</td></tr><tr><td>5TRV _long</td><td>0.00</td><td>0.00</td><td>0.19</td><td>0.00</td><td>0.37</td><td>0.08</td><td>0.19</td><td>0.00</td><td>0.07</td></tr><tr><td>5TRV _med</td><td>0.00</td><td>0.00</td><td>0.16</td><td>0.03</td><td>0.24</td><td>0.07</td><td>0.16</td><td>0.02</td><td>0.19</td></tr><tr><td>5TRV_short</td><td>0.00</td><td>0.00</td><td>0.01</td><td>0.07</td><td>0.04</td><td>0.10</td><td>0.01</td><td>0.03</td><td>0.11</td></tr><tr><td>5WN9</td><td>0.00</td><td>0.00</td><td>0.02</td><td>0.00</td><td>0.00</td><td>0.20</td><td>0.00</td><td>0.00</td><td>0.00</td></tr><tr><td>5YUI</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.02</td><td>0.00</td><td>0.00</td><td>0.00</td><td>0.00</td></tr><tr><td>6E6R_long</td><td>0.01</td><td>0.65</td><td>0.07</td><td>0.91</td><td>0.86</td><td>0.92</td><td>0.04</td><td>0.78</td><td>1.00</td></tr><tr><td>6E6R_med</td><td>0.03</td><td>0.94</td><td>0.24</td><td>0.93</td><td>0.89</td><td>0.88</td><td>0.14</td><td>0.77</td><td>0.97</td></tr><tr><td>6E6R_short</td><td>0.07</td><td>0.87</td><td>0.09</td><td>0.86</td><td>0.39</td><td>0.78</td><td>0.06</td><td>0.64</td><td>0.99</td></tr><tr><td>6EXZ_long</td><td>0.00</td><td>0.01</td><td>0.32</td><td>0.61</td><td>0.76</td><td>0.63</td><td>0.13</td><td>0.44</td><td>0.95</td></tr><tr><td>6EXZ_med</td><td>0.00</td><td>0.00</td><td>0.31</td><td>0.66</td><td>0.49</td><td>0.63</td><td>0.31</td><td>0.55</td><td>0.96</td></tr><tr><td>6EXZ_short</td><td>0.00</td><td>0.00</td><td>0.31</td><td>0.66</td><td>0.39</td><td>0.41</td><td>0.28</td><td>0.58</td><td>0.87</td></tr><tr><td>7MRX_long</td><td>0.00</td><td>0.02</td><td>0.36</td><td>0.23</td><td>0.09</td><td>0.32</td><td>0.37</td><td>0.20</td><td>0.73</td></tr><tr><td>7MRX_med</td><td>0.00</td><td>0.31</td><td>0.65</td><td>0.28</td><td>0.11</td><td>0.31</td><td>0.59</td><td>0.22</td><td>0.70</td></tr><tr><td>7MRX. _short</td><td>0.00</td><td>0.34</td><td>0.68</td><td>0.26</td><td>0.02</td><td>0.41</td><td>0.74</td><td>0.24</td><td>0.88</td></tr><tr><td>pass rate</td><td>7/24</td><td>11/24</td><td>21/24</td><td>18/24</td><td>20/24</td><td>20/24</td><td>20/24</td><td>18/24</td><td>19/24</td></tr><tr><td>avg. success rate</td><td>0.04</td><td>0.19</td><td>0.25</td><td>0.35</td><td>0.40</td><td>0.42</td><td>0.18</td><td>0.29</td><td>0.53</td></tr></table></table></figure><blockquote><p>Table 2 benchmarks the performance of DPLM-2 against other methods on unconditional protein generation tasks, evaluating various metrics across different protein lengths and generation approaches.</p></blockquote></details><h3 class="relative group">Full paper<div id=full-paper class=anchor></div><span class="absolute top-0 w-6 transition-opacity opacity-0 ltr:-left-6 rtl:-right-6 not-prose group-hover:opacity-100"><a class="group-hover:text-primary-300 dark:group-hover:text-neutral-700" style=text-decoration-line:none!important href=#full-paper aria-label=Anchor>#</a></span></h3><div id=gallery-db14043eba4a64e6210884c4bab6aebe class=gallery><img src=paper_images/1.png class="grid-w50 md:grid-w33 xl:grid-w25">
<img src=paper_images/2.png class="grid-w50 md:grid-w33 xl:grid-w25">
<img src=paper_images/3.png class="grid-w50 md:grid-w33 xl:grid-w25">
<img src=paper_images/4.png class="grid-w50 md:grid-w33 xl:grid-w25">
<img src=paper_images/5.png class="grid-w50 md:grid-w33 xl:grid-w25">
<img src=paper_images/6.png class="grid-w50 md:grid-w33 xl:grid-w25">
<img src=paper_images/7.png class="grid-w50 md:grid-w33 xl:grid-w25">
<img src=paper_images/8.png class="grid-w50 md:grid-w33 xl:grid-w25">
<img src=paper_images/9.png class="grid-w50 md:grid-w33 xl:grid-w25">
<img src=paper_images/10.png class="grid-w50 md:grid-w33 xl:grid-w25">
<img src=paper_images/11.png class="grid-w50 md:grid-w33 xl:grid-w25">
<img src=paper_images/12.png class="grid-w50 md:grid-w33 xl:grid-w25">
<img src=paper_images/13.png class="grid-w50 md:grid-w33 xl:grid-w25">
<img src=paper_images/14.png class="grid-w50 md:grid-w33 xl:grid-w25">
<img src=paper_images/15.png class="grid-w50 md:grid-w33 xl:grid-w25">
<img src=paper_images/16.png class="grid-w50 md:grid-w33 xl:grid-w25">
<img src=paper_images/17.png class="grid-w50 md:grid-w33 xl:grid-w25">
<img src=paper_images/18.png class="grid-w50 md:grid-w33 xl:grid-w25">
<img src=paper_images/19.png class="grid-w50 md:grid-w33 xl:grid-w25"></div></div><section class="flex flex-row flex-wrap justify-center pt-4 text-xl"><a target=_blank class="m-1 rounded bg-neutral-300 p-1.5 text-neutral-700 hover:bg-primary-500 hover:text-neutral dark:bg-neutral-700 dark:text-neutral-300 dark:hover:bg-primary-400 dark:hover:text-neutral-800" href="https://www.linkedin.com/shareArticle?mini=true&amp;url=https://deep-diver.github.io/ai-paper-reviewer/paper-reviews/2410.13782/&amp;title=DPLM-2:%20A%20Multimodal%20Diffusion%20Protein%20Language%20Model" title="Share on LinkedIn" aria-label="Share on LinkedIn"><span class="relative block icon"><svg viewBox="0 0 448 512"><path fill="currentcolor" d="M416 32H31.9C14.3 32 0 46.5.0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6.0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3.0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2.0 38.5 17.3 38.5 38.5.0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6.0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2.0 79.7 44.3 79.7 101.9V416z"/></svg>
</span></a><a target=_blank class="m-1 rounded bg-neutral-300 p-1.5 text-neutral-700 hover:bg-primary-500 hover:text-neutral dark:bg-neutral-700 dark:text-neutral-300 dark:hover:bg-primary-400 dark:hover:text-neutral-800" href="https://twitter.com/intent/tweet/?url=https://deep-diver.github.io/ai-paper-reviewer/paper-reviews/2410.13782/&amp;text=DPLM-2:%20A%20Multimodal%20Diffusion%20Protein%20Language%20Model" title="Tweet on Twitter" aria-label="Tweet on Twitter"><span class="relative block icon"><svg viewBox="0 0 512 512"><path fill="currentcolor" d="M389.2 48h70.6L305.6 224.2 487 464H345L233.7 318.6 106.5 464H35.8L200.7 275.5 26.8 48H172.4L272.9 180.9 389.2 48zM364.4 421.8h39.1L151.1 88h-42L364.4 421.8z"/></svg>
</span></a><a target=_blank class="m-1 rounded bg-neutral-300 p-1.5 text-neutral-700 hover:bg-primary-500 hover:text-neutral dark:bg-neutral-700 dark:text-neutral-300 dark:hover:bg-primary-400 dark:hover:text-neutral-800" href="mailto:?body=https://deep-diver.github.io/ai-paper-reviewer/paper-reviews/2410.13782/&amp;subject=DPLM-2:%20A%20Multimodal%20Diffusion%20Protein%20Language%20Model" title="Send via email" aria-label="Send via email"><span class="relative block icon"><svg viewBox="0 0 512 512"><path fill="currentcolor" d="M207.8 20.73c-93.45 18.32-168.7 93.66-187 187.1-27.64 140.9 68.65 266.2 199.1 285.1 19.01 2.888 36.17-12.26 36.17-31.49l1e-4-.6631c0-15.74-11.44-28.88-26.84-31.24-84.35-12.98-149.2-86.13-149.2-174.2.0-102.9 88.61-185.5 193.4-175.4 91.54 8.869 158.6 91.25 158.6 183.2v16.16c0 22.09-17.94 40.05-40 40.05s-40.01-17.96-40.01-40.05v-120.1c0-8.847-7.161-16.02-16.01-16.02l-31.98.0036c-7.299.0-13.2 4.992-15.12 11.68-24.85-12.15-54.24-16.38-86.06-5.106-38.75 13.73-68.12 48.91-73.72 89.64-9.483 69.01 43.81 128 110.9 128 26.44.0 50.43-9.544 69.59-24.88 24 31.3 65.23 48.69 109.4 37.49C465.2 369.3 496 324.1 495.1 277.2V256.3c0-149.2-133.9-265.632-287.3-235.57zM239.1 304.3c-26.47.0-48-21.56-48-48.05s21.53-48.05 48-48.05 48 21.56 48 48.05-20.6 48.05-48 48.05z"/></svg></span></a></section></div><script>var oid="views_paper-reviews/2410.13782/index.md",oid_likes="likes_paper-reviews/2410.13782/index.md"</script><script type=text/javascript src=/ai-paper-reviewer/js/page.min.0860cf4e04fa2d72cc33ddba263083464d48f67de06114529043cb4623319efed4f484fd7f1730df5abea0e2da6f3538855634081d02f2d6e920b956f063e823.js integrity="sha512-CGDPTgT6LXLMM926JjCDRk1I9n3gYRRSkEPLRiMxnv7U9IT9fxcw31q+oOLabzU4hVY0CB0C8tbpILlW8GPoIw=="></script></section><footer class="pt-8 max-w-prose print:hidden"><div class=pt-8><hr class="border-dotted border-neutral-300 dark:border-neutral-600"><div class="flex justify-between pt-3"><span><a class="flex group mr-3" href=/ai-paper-reviewer/paper-reviews/2410.13925/><span class="mr-3 text-neutral-700 group-hover:text-primary-600 ltr:inline rtl:hidden dark:text-neutral dark:group-hover:text-primary-400">&larr;</span>
<span class="ml-3 text-neutral-700 group-hover:text-primary-600 ltr:hidden rtl:inline dark:text-neutral dark:group-hover:text-primary-400">&rarr;</span>
<span class="flex flex-col"><span class="mt-[0.1rem] leading-6 group-hover:underline group-hover:decoration-primary-500">FiTv2: Scalable and Improved Flexible Vision Transformer for Diffusion Model</span>
<span class="mt-[0.1rem] text-xs text-neutral-500 dark:text-neutral-400"><time datetime=2024-10-17T00:00:00+00:00>17 October 2024</time>
</span></span></a></span><span><a class="flex text-right group ml-3" href=/ai-paper-reviewer/paper-reviews/2410.13674/><span class="flex flex-col"><span class="mt-[0.1rem] leading-6 group-hover:underline group-hover:decoration-primary-500">Diffusion Curriculum: Synthetic-to-Real Generative Curriculum Learning via Image-Guided Diffusion</span>
<span class="mt-[0.1rem] text-xs text-neutral-500 dark:text-neutral-400"><time datetime=2024-10-17T00:00:00+00:00>17 October 2024</time>
</span></span><span class="ml-3 text-neutral-700 group-hover:text-primary-600 ltr:inline rtl:hidden dark:text-neutral dark:group-hover:text-primary-400">&rarr;</span>
<span class="mr-3 text-neutral-700 group-hover:text-primary-600 ltr:hidden rtl:inline dark:text-neutral dark:group-hover:text-primary-400">&larr;</span></a></span></div></div><div class=pt-3><hr class="border-dotted border-neutral-300 dark:border-neutral-600"><div class=pt-3><script src=https://utteranc.es/client.js repo=pmnxis/pmnxis.github.io issue-term=pathname label=Comment theme=dark-blue crossorigin=anonymous async></script></div></div></footer></article><div id=top-scroller class="pointer-events-none absolute top-[110vh] bottom-0 w-12 ltr:right-0 rtl:left-0"><a href=#the-top class="pointer-events-auto sticky top-[calc(100vh-5.5rem)] flex h-12 w-12 mb-16 items-center justify-center rounded-full bg-neutral/50 text-xl text-neutral-700 hover:text-primary-600 dark:bg-neutral-800/50 dark:text-neutral dark:hover:text-primary-400" aria-label="Scroll to top" title="Scroll to top">&uarr;</a></div></main><footer id=site-footer class="py-10 print:hidden"><nav class="flex flex-row pb-4 text-base font-medium text-neutral-500 dark:text-neutral-400"><ul class="flex flex-col list-none sm:flex-row"><li class="flex mb-1 ltr:text-right rtl:text-left sm:mb-0 ltr:sm:mr-7 ltr:sm:last:mr-0 rtl:sm:ml-7 rtl:sm:last:ml-0"><a class="decoration-primary-500 hover:underline hover:decoration-2 hover:underline-offset-2 flex items-center" href=/ai-paper-reviewer/tags/ title>Tags</a></li></ul></nav><div class="flex items-center justify-between"><p class="text-sm text-neutral-500 dark:text-neutral-400">&copy;
2024
AI Paper Reviews by AI</p><p class="text-xs text-neutral-500 dark:text-neutral-400">Powered by <a class="hover:underline hover:decoration-primary-400 hover:text-primary-500" href=https://gohugo.io/ target=_blank rel="noopener noreferrer">Hugo</a> & <a class="hover:underline hover:decoration-primary-400 hover:text-primary-500" href=https://blowfish.page/ target=_blank rel="noopener noreferrer">Blowfish</a></p></div><script>mediumZoom(document.querySelectorAll("img:not(.nozoom)"),{margin:24,background:"rgba(0,0,0,0.5)",scrollOffset:0})</script><script type=text/javascript src=/ai-paper-reviewer/js/process.min.ee03488f19c93c2efb199e2e3014ea5f3cb2ce7d45154adb3399a158cac27ca52831db249ede5bb602700ef87eb02434139de0858af1818ab0fb4182472204a4.js integrity="sha512-7gNIjxnJPC77GZ4uMBTqXzyyzn1FFUrbM5mhWMrCfKUoMdsknt5btgJwDvh+sCQ0E53ghYrxgYqw+0GCRyIEpA=="></script></footer><div id=search-wrapper class="invisible fixed inset-0 flex h-screen w-screen cursor-default flex-col bg-neutral-500/50 p-4 backdrop-blur-sm dark:bg-neutral-900/50 sm:p-6 md:p-[10vh] lg:p-[12vh]" data-url=https://deep-diver.github.io/ai-paper-reviewer/ style=z-index:500><div id=search-modal class="flex flex-col w-full max-w-3xl min-h-0 mx-auto border rounded-md shadow-lg top-20 border-neutral-200 bg-neutral dark:border-neutral-700 dark:bg-neutral-800"><header class="relative z-10 flex items-center justify-between flex-none px-2"><form class="flex items-center flex-auto min-w-0"><div class="flex items-center justify-center w-8 h-8 text-neutral-400"><span class="relative block icon"><svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="search" class="svg-inline--fa fa-search fa-w-16" role="img" viewBox="0 0 512 512"><path fill="currentcolor" d="M505 442.7 405.3 343c-4.5-4.5-10.6-7-17-7H372c27.6-35.3 44-79.7 44-128C416 93.1 322.9.0 208 0S0 93.1.0 208s93.1 208 208 208c48.3.0 92.7-16.4 128-44v16.3c0 6.4 2.5 12.5 7 17l99.7 99.7c9.4 9.4 24.6 9.4 33.9.0l28.3-28.3c9.4-9.4 9.4-24.6.1-34zM208 336c-70.7.0-128-57.2-128-128 0-70.7 57.2-128 128-128 70.7.0 128 57.2 128 128 0 70.7-57.2 128-128 128z"/></svg></span></div><input type=search id=search-query class="flex flex-auto h-12 mx-1 bg-transparent appearance-none focus:outline-dotted focus:outline-2 focus:outline-transparent" placeholder=Search tabindex=0></form><button id=close-search-button class="flex items-center justify-center w-8 h-8 text-neutral-700 hover:text-primary-600 dark:text-neutral dark:hover:text-primary-400" title="Close (Esc)">
<span class="relative block icon"><svg viewBox="0 0 320 512"><path fill="currentcolor" d="M310.6 361.4c12.5 12.5 12.5 32.75.0 45.25C304.4 412.9 296.2 416 288 416s-16.38-3.125-22.62-9.375L160 301.3 54.63 406.6C48.38 412.9 40.19 416 32 416S15.63 412.9 9.375 406.6c-12.5-12.5-12.5-32.75.0-45.25l105.4-105.4L9.375 150.6c-12.5-12.5-12.5-32.75.0-45.25s32.75-12.5 45.25.0L160 210.8l105.4-105.4c12.5-12.5 32.75-12.5 45.25.0s12.5 32.75.0 45.25l-105.4 105.4L310.6 361.4z"/></svg></span></button></header><section class="flex-auto px-2 overflow-auto"><ul id=search-results></ul></section></div></div></div></body></html>